contrib/examples/apiserver/binding.yaml

```diff
@@ -3,4 +3,9 @@ kind: Binding
 metadata:
   name: test-binding
 spec:
+  instanceRef:
+    name: test-instance
   secretName: my-secret
+  configMapName: my-configmap
```

Comment:
Until we get changes to the OSB API and can make a distinction between secret and configMap fields, it seems like we should put them all in the secret.